Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -314,23 +324,46 @@ void OpenALStream::SoundLoop()
     // many silence samples.  These do not need to be timestretched.
     if (rate > 0.10)
     {
-      soundTouch.setTempo(rate);
-      if (rate > 10)
+      if (use_timestretching)
+      {
+        soundTouch.setTempo(rate);
+      }
+      else
+      {
+        alSourcef(ui_source, AL_PITCH, static_cast<ALfloat>(rate));
```

Comment:
Er, what do you mean? They should both be resampling by exactly the same amount. If you don't mind recording a video, that would be helpful.